doc/api/stream.md

```diff
@@ -3181,6 +3181,8 @@ changes:
       If no value is provided, the size will be `1` for all the chunks.
       * `chunk` {any}
       * Returns: {number}
+  * `type` {string} Must be 'bytes' or undefined.
+    If `type` is set to 'bytes', the `strategy` option is ignored
```

Comment:
Hmm... when `type` is `bytes`, only the `strategy.size` option should be ignored. The `strategy.highWaterMark` is still relevant and should not be ignored.